### DISCUSSION TITLE
Exporting a flow for tools like newman or anything a azure pipeline or github action can start

### DISCUSSION THREAD

**Discussion by Poolitzer80** [postmanlabs/postman-flows#126] [Ideas]
For CI/CD reasons it would be awesome, if a designed flow could be exported somehow to a script or file, newman e.G. could call.

**Comment by samay-v** [comment]
Hey @Poolitzer80 
This thread talks about this https://github.com/postmanlabs/postman-flows/discussions/19#discussioncomment-1458902
